src/Swashbuckle.AspNetCore.SwaggerGen/DependencyInjection/SwaggerGenOptionsExtensions.cs

```diff
@@ -195,6 +195,33 @@ public static void AddServer(this SwaggerGenOptions swaggerGenOptions, OpenApiSe
             swaggerGenOptions.MapType(typeof(T), schemaFactory);
         }
 
+        /// <summary>
+        /// Provide a custom mapping, for a given type, to the Swagger-flavored JSONSchema
```

Comment:
Unless "JSONSchema" is a specific term I've not heard before:

```suggestion
        /// Provides a custom mapping, for a given type, to the Swagger-flavored JSON schema
```